### ISSUE TITLE
AWS Lambda support missing for dotnetcore2.1. 

### ISSUE THREAD

**Issue by CasperSkydt** [terraform-providers/terraform-provider-aws#5148]
Is getting the following error:

expected runtime to be one of [dotnetcore1.0 dotnetcore2.0 go1.x java8 nodejs4.3 nodejs4.3-edge nodejs6.10 nodejs8.10 python2.7 python3.6], got dotnetcore2.1

Not being familiar with go, i'm guessing this is the problem though:

https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_lambda_function.go#L106

Its missing dotnetcore2.1

https://aws.amazon.com/about-aws/whats-new/2018/06/lambda-supports-dotnetcore-twopointone/

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### New or Affected Resource(s)

* aws_XXXXX

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000

**Comment by bflad** [comment]
Pull request submitted to add `dotnetcore2.1` to the `runtime` argument validation: #5150

**Comment by bflad** [comment]
This has been merged in and will release with version 1.27.0 of the AWS provider, likely later today.

**Comment by CasperSkydt** [comment]
That was really fast, awesome work, and thanks for the help.

**Comment by bflad** [comment]
This has been released in [version 1.27.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1270-july-11-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
